packages/core/analytics/core-analytics-browser-internal/src/track_performance_measure_entries.ts

```diff
@@ -18,6 +25,19 @@ export function trackPerformanceMeasureEntries(analytics: AnalyticsClient, isDev
       if (entry.entryType === 'measure' && entry.detail?.type === 'kibana:performance') {
         const target = entry?.name;
         const duration = entry.duration;
+        const customMetrics = Object.keys(entry.detail?.customMetrics ?? {}).reduce(
+          (acc, metric) => {
+            if (ALLOWED_CUSTOM_METRICS_KEYS_VALUES.includes(metric)) {
```

Comment:
Do you need to check this since use the type here: https://github.com/elastic/kibana/pull/189115/files#diff-533ff799a897d207273d4b8161d1cbc73c72590b0f3e6d9c80fe3410c5dbaac9R16

but anyways it doesn't hurt to check it though